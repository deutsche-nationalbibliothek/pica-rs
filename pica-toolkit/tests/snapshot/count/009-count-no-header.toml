bin.name = "pica"
args = "count -s --tsv --no-header"
status = "success"
stderr = ""
