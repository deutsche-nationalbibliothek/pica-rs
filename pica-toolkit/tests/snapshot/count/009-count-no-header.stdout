1	28	56
